testing/run_tests.sh

```diff
@@ -187,26 +106,18 @@ check_kvm() {
 
 usage() {
     echo "Usage: ./run_tests.sh [-m] [-d gs_bucket] [-j jobs]"
```

Comment:
This option just got removed.
```suggestion
    echo "Usage: ./run_tests.sh  [-d gs_bucket] [-j jobs]"
```